web-components/src/components/toggle-switch/ToggleSwitch.ts

```diff
@@ -38,27 +45,51 @@ export namespace ToggleSwitch {
     }
 
     render() {
-      return html`
-        <div
-          class="md-input-container md-toggle-switch  ${classMap(this.toggleSwitchClassMap)}"
-          @click=${this.handleClick}
-        >
-          <input
-            type="checkbox"
-            class="md-toggle-switch__input"
-            aria-label="${this.label}"
-            .id=${this.htmlId}
-            ?checked=${this.checked}
-            ?disabled=${this.disabled}
-            ?autofocus=${this.autofocus}
-            tabindex=${this.disabled ? -1 : 0}
-          />
-          <md-label .htmlFor=${this.htmlId} class="md-toggle-switch__label">
-            <span class="md-toggle-switch__label__container"></span>
-            <slot></slot>
-          </md-label>
-        </div>
-      `;
+      if (this.align == "right") {
+        return html`
+          <div
+            class="md-input-container md-toggle-switch  ${classMap(this.toggleSwitchClassMap)}"
+            @click=${this.handleClick}
+          >
+            <input
```

Comment:
if and else condition is almost similar, can you please refactor the code to extract reusable code and reuse in both the condition?

Reply:
Done